src/main/scala/it/pagopa/pdnd/interop/uservice/partymanagement/api/impl/PartyApiServiceImpl.scala

```diff
@@ -149,13 +144,15 @@ class PartyApiServiceImpl(
           .fold(
             _ => {
               logger.error("Error adding attributes to organization {} - Bad request", organizationId)
-              addOrganizationAttributes404(problemOf(StatusCodes.BadRequest, "0005"))
+              addOrganizationAttributes404(problemOf(StatusCodes.BadRequest, AddAttributesBadRequest))
             },
             organization => addOrganizationAttributes200(organization)
           )
       case statusReply =>
         logger.error("Error adding attributes to organization {} - Not found", organizationId)
-        addOrganizationAttributes404(problemOf(StatusCodes.NotFound, "0006", statusReply.getError))
+        addOrganizationAttributes404(
+          problemOf(StatusCodes.NotFound, AddAttributesError(statusReply.getError.getMessage))
```

Comment:
is it ok here to leak the error message?

Reply:
it should not. We could pass a generic string message here.